.github/workflows/check.yaml

```diff
@@ -0,0 +1,21 @@
+name: "Lint PRs"
+
+on: pull_request
+
+jobs:
```

Comment:
It wasn't that obvious to me where to look to understand what tooling is being used and what commit rules it is going to try to enforce. Could you add a block comment with references so that newbies like me can read the right sources?

Reply:
Added something